pkg/api/collectorcontrollerv1alpha1/types.go

```diff
@@ -597,10 +597,21 @@ type LabelsMask struct {
 	// Extensions defines the mask for user defined labels pulled from object metadata.
 	Extensions ExtensionsLabelMask `json:"extensions" yaml:"extensions"`
 
+	// Filters excludes metrics from aggregation based on some conditions
+	Filters []Filter `json:"filters" yaml:"filters"`
```

Comment:
What stops the user from setting two filters, one saying that `label1` should be excluded and then `label1` should be included?

What's the expected behavior? I think it's basically the first rule that matches the label name applies, but it'd be good to test and document.

Reply:
It would exclude all metrics since both filters would apply.  Essentially the same as

```go
b := true
if b && !b {
  // do something
}

```